### PR TITLE
squash on build

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: hack/build.sh [hunk 1]
```diff
@@ -9,6 +9,23 @@ declare -a VERSIONS=(9.2)
 OS=$1
 VERSION=$2
 
+function squash {	
+	base=$(grep FROM Dockerfile | awk '{print $2}')
```
Comment (mfojtik) [inline hack/build.sh]: grep "^FROM"

File: hack/build.sh [hunk 2]
```diff
@@ -9,6 +9,23 @@ declare -a VERSIONS=(9.2)
...
+	# install the docker layer squashing tool
+	easy_install --user pip
+	rm -rf docker-scripts
+	git clone https://github.com/goldmann/docker-scripts/
```
Comment (mfojtik) [inline hack/build.sh]: git clone --branch 0.3.0

File: hack/build.sh [hunk 3]
```diff
@@ -9,6 +9,23 @@ declare -a VERSIONS=(9.2)
...
+	pushd docker-scripts
+	git checkout 0.3.0
+	~/.local/bin/pip install --force --user .
```
Comment (mfojtik) [inline hack/build.sh]: s/~/$HOME/

**Comment by bparees** [comment]
@mfojtik thanks, updated here and in the 7 other pulls :)

**Pull request merged**